docs/modules/ROOT/pages/how-tos/testing/build/index.adoc

```diff
@@ -1 +1,56 @@
-= Build-time tests
\ No newline at end of file
+= Build-time tests
+
+This document covers the build-time tests that {ProductName} runs as part of its component build pipeline. These build-time tests automatically check all application images to inform you if they're up-to-date, correctly formatted, and protected from security vulnerabilities.
+
+NOTE: These tests are non-blocking and do not prevent the pipeline from proceeding if issues are found.
+
+The component build pipeline for {ProductName} include various types of build-time tests, conducted using link:https://tekton.dev/docs/pipelines/tasks/#overview[Tekton tasks]. For validating container information, the pipeline employs https://www.conftest.dev/[Conftest]. 
+
+The table below outlines the default build-time tests:
+
+.Deprecated image checks
+|===
+|Test name |Description |Failure message
+
+|image_repository_deprecated |Deprecated images are no longer maintained, leading to unresolved security vulnerabilities. | The container image must not be built from a repository  marked as 'Deprecated' in COMET
+|===
+
+.Unsigned RPM check
+|===
+|Test name |Description |Failure message
+
+|image_unsigned_rpms |Packages signed with Red Hat's secure signing server adheres to stringent policies and procedures. |Alerts the user for any unsigned RPMs. Found following unsigned rpms(nvra):
+|===
+
+.Source code check
+|===
+|Test name |Description |Failure message
+
+|sast-snyk-check |Scans and analyze source code or compiled versions of code to help find security flaws. | <tbd>
+|===
+
+You can disable the default build-time tests by setting the `skip-checks` parameter to `true`
+
+[source,yaml]
+----
+- description: Skip checks against built image
+  name: skip-checks
+  type: string
+  default: "false"
+----
+
+.Security checks
```

Comment:
```suggestion
== Security checks
```

Reply:
nevermind, it's not markdown :D

Reply:
or maybe yes, we should have a separate section for tests

Reply:
yes, its asciidoc
renders the security tests within a separate table like this:
![image](https://github.com/konflux-ci/docs/assets/43273420/1227ef96-eb6e-41a4-b455-5d0f539b385f)

Reply:
![image](https://github.com/konflux-ci/docs/assets/20207783/272f13c6-23a1-4896-be33-173e5c47b3e5)
This should go after the tables, as it;s valid for all tests

Reply:
done!

Reply:
https://github.com/konflux-ci/konflux-test/blob/a41b880d19feb7fa56bcce446c646122df4f181c/test/utils.sh#L13 it will have this default output